src/main/java/uk/ac/ebi/embl/flatfile/reader/embl/SQReader.java

```diff
@@ -15,25 +15,43 @@
  ******************************************************************************/
 package uk.ac.ebi.embl.flatfile.reader.embl;
 
-import uk.ac.ebi.embl.flatfile.reader.SingleLineBlockReader;
+import org.apache.commons.lang3.StringUtils;
 import uk.ac.ebi.embl.flatfile.EmblTag;
 import uk.ac.ebi.embl.flatfile.reader.LineReader;
+import uk.ac.ebi.embl.flatfile.reader.SingleLineBlockReader;
 
-/** Reader for the SQ line.
+import java.util.Arrays;
+
+/**
+ * Reader for the SQ line. Capture coverage values
  */
 public class SQReader extends SingleLineBlockReader {
 
-	public SQReader(LineReader lineReader) {
-		super(lineReader); //, ConcatenateType.CONCATENATE_NOSPACE);
-	}
+    public SQReader(LineReader lineReader) {
+        super(lineReader); //, ConcatenateType.CONCATENATE_NOSPACE);
+    }
+
+    @Override
+    public String getTag() {
+        return EmblTag.SQ_TAG;
+    }
 
-	@Override
-	public String getTag() {
-		return EmblTag.SQ_TAG;
-	}
-		
-	@Override
-	protected void read(String block) {
-		return;
-	}
+    @Override
+    protected void read(String block) {
+        // SQ   Sequence 315242 BP; 87432 A; 72431 C; 71123 G; 84256 T; 0 other;
+        if (StringUtils.isNotBlank(block)) {
+            try {
+                Arrays.stream(StringUtils.stripAll(
+                        StringUtils.split(
+                                StringUtils.stripStart(block, "SQ   Sequence"), ";")))
+                        .forEach(s -> {
+                            final String[] split = StringUtils.split(s, " ");
+                            entry.getSequenceCoverage().put(split[1], Long.valueOf(split[0]));
+                        });
+            } catch (Exception e) {
+                // fail silently
```

Comment:
not sure if its a good idea to do nothing with the exception. can we somehow convey this failure to ValidationResult object?

Reply:
as the SQ line doesn't have a strict format, this silent fail ensures existing flows/old files won't fail.